Evaluating_pipeline.md

```diff
@@ -0,0 +1,192 @@
+
+Some software required by the remapping pipeline are also assumed to be installed and in the `PATH`
+
+## Preparing GiaB datasets 
+
+Set `GIAB_DIR` to points to the directory where the variants will be stored
+
+|:warning: This can be skipped if the datasets have been prepared before.|
+|---|
+
+The variant remapping pipeline was run for on two variant datasets that were download from GiaB website. 
+The variant data is here:
+
+ftp://ftp-trace.ncbi.nlm.nih.gov/giab/ftp/release/NA12878_HG001/NISTv3.3.2/
+
+The reference genomes used for calling the GiaB variant are defined in
+ [the FAQ point 5](https://www.nist.gov/programs-projects/faqs-genome-bottle)
+
+```
+GRCh37 reference with decoy:
+ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/technical/reference/phase2_reference_assembly_sequence/hs37d5.fa.gz
+
+GRCh38 reference with no ALT loci: 
+ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000/001/405/GCA_000001405.15_GRCh38/seqs_for_alignment_pipelines.ucsc_ids/GCA_000001405.15_GRCh38_no_alt_analysis_set.fna.gz
+```
+Prepare directory structure
+
+```bash
+mkdir -p ${GIAB_DIR}/NA12878/GRCh37
+mkdir -p ${GIAB_DIR}NA12878/GRCh38
+```
+
+Download the variant datasets 
+
+
+```bash
+cd ${GIAB_DIR}/NA12878/GRCh37
+wget ftp://ftp-trace.ncbi.nlm.nih.gov/giab/ftp/release/NA12878_HG001/NISTv3.3.2/GRCh37/HG001_GRCh37_GIAB_highconf_CG-IllFB-IllGATKHC-Ion-10X-SOLID_CHROM1-X_v.3.3.2_highconf_PGandRTGphasetransfer.vcf.gz
+wget ftp://ftp-trace.ncbi.nlm.nih.gov/giab/ftp/release/NA12878_HG001/NISTv3.3.2/GRCh37/HG001_GRCh37_GIAB_highconf_CG-IllFB-IllGATKHC-Ion-10X-SOLID_CHROM1-X_v.3.3.2_highconf_PGandRTGphasetransfer.vcf.gz.tbi
+wget ftp://ftp-trace.ncbi.nlm.nih.gov/giab/ftp/release/NA12878_HG001/NISTv3.3.2/GRCh37/HG001_GRCh37_GIAB_highconf_CG-IllFB-IllGATKHC-Ion-10X-SOLID_CHROM1-X_v.3.3.2_highconf_nosomaticdel.bed
+```
+
+```bash
+cd ${GIAB_DIR}/NA12878/GRCh38
+wget ftp://ftp-trace.ncbi.nlm.nih.gov/giab/ftp/release/NA12878_HG001/NISTv3.3.2/GRCh38/HG001_GRCh38_GIAB_highconf_CG-IllFB-IllGATKHC-Ion-10X-SOLID_CHROM1-X_v.3.3.2_highconf_PGandRTGphasetransfer.vcf.gz
+wget ftp://ftp-trace.ncbi.nlm.nih.gov/giab/ftp/release/NA12878_HG001/NISTv3.3.2/GRCh38/HG001_GRCh38_GIAB_highconf_CG-IllFB-IllGATKHC-Ion-10X-SOLID_CHROM1-X_v.3.3.2_highconf_PGandRTGphasetransfer.vcf.gz.tbi
+wget ftp://ftp-trace.ncbi.nlm.nih.gov/giab/ftp/release/NA12878_HG001/NISTv3.3.2/GRCh38/HG001_GRCh38_GIAB_highconf_CG-IllFB-IllGATKHC-Ion-10X-SOLID_CHROM1-X_v.3.3.2_highconf_nosomaticdel_noCENorHET7.bed
+```
+
+|:warning: GRCh38 is not annotated -> Need to annotate it with dbsnp data|
+|---|
+
+Download dbSNPs vcf to get rsids
+```
+wget ftp://ftp.ncbi.nih.gov/snp/organisms/human_9606/VCF/common_all_20180418.vcf.gz
+```
+
+|:warning: dbSNP chromosome name are 1, 2, 3, ... where hs37d5 is chr1, chr2, chr3 ...  We'll prefix all chromosome name with chr|
+|---|
+
+```bash
+zcat common_all_20180418.vcf.gz | awk '{if (/^#/){print } else{print "chr"$0}}' | bgzip -c > common_all_20180418_with_chr.vcf.gz
```

Comment:
Are there only chromosome numbers and no contig names in the first column?

Reply:
They were only chromosome numbers, not even an MT. Which made my life a bit easier